File: Transfers/TicketTransfers.cs
namespace Kanban.Transfers;

public record TicketCreateRequest(
	string Title,
	string? Description
);

public record TicketUpdateRequest(
	int? Position,
	string? Title,
	string? Description,
	string? ColumnId
);

public record TicketDetailResponse(
	string Id,
	string Number,
	int Position,
	string Title,
	string Description,
	string ColumnId
);

File: Frontend/src/transfers/ticketTransfers.ts
export interface TicketCreateRequest {
	title: string,
	description?: string
}

export interface TicketUpdateRequest {
	position?: number,
	title?: string,
	description?: string,
	columnId?: string
}

export interface TicketStructureRequest {
	id: string
}

export interface TicketSimpleResponse {
	id: string,
	position: number,
	title: string,
	number: string
}

export interface TicketDetailResponse {
	id: string,
	number: string,
	position: number,
	title: string,
	description: string,
	columnId: string
}